README.md

```diff
@@ -43,19 +48,9 @@ You can also include it directly from a CDN:
 <script src="https://cdn.jsdelivr.net/npm/@turf/turf@6/turf.min.js"></script>
 ```
 
-You can create light-weight turf builds with only the functions you need using the [turfjs-builder UI](https://turfjs-builder.herokuapp.com/) or using browserify as described below.
```

Comment:
turfjs-builder seems to be using an old version of turf and I cannot really recommend it any more.

Reply:
Yeah I think I'll archive that bad boy